### PR TITLE
i18n add Occitan language

### PR DESCRIPTION
**Pull request by Mejans**
Hello,
I really don't know about flutter and stuff, so I tried to do the same as the FR file I found.
Best regards

### CODE REVIEW

File: ubuntu_desktop_installer/lib/l10n/intl_oc.arb [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  "@Continue": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "Restart": "Reaviar",
+  "@Restart": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "Try or install": "Tornar ensajar o installar",
```
Comment (oSoMoN) [inline ubuntu_desktop_installer/lib/l10n/intl_oc.arb:23]: I don't speak Occitan (but I do speak Catalan and French, so I can understand most of it). This "Tornar" looks out of place, is this correct?

File: ubuntu_desktop_installer/lib/l10n/intl_oc.arb [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  "@You can try Ubuntu without making any changes to your computer.": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "Install Ubuntu": "Installar Ubuntu",
+  "@Install Ubuntu": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "Install Ubuntu alongside (or instead of) your current operating system. This shouldn't take too long.": "Installer Ubuntu à côté (ou en remplacement) de votre système d’exploitation actuel. Ceci ne devrait pas prendre trop longtemps.",
```
Comment (oSoMoN) [inline ubuntu_desktop_installer/lib/l10n/intl_oc.arb:53]: You forgot to translate this one, it's copy/pasted from the French translations.

**Review by oSoMoN — CHANGES_REQUESTED** [review]
Thanks for these translations, they look fine to me.

See a couple of comments inline.

Note that you would also need to re-generate the corresponding files in `lib/l10n` per the instructions in README.md.

File: ubuntu_desktop_installer/lib/l10n/intl_oc.arb (hunk 1 above)
Comment (Mejans) [reply ubuntu_desktop_installer/lib/l10n/intl_oc.arb:23]: yes right, I had in mind "retry" somehow :)

**Comment by Mejans** [comment]
It's nice to have someone to check :)
About the generation, I don't know how to do this =/

**Comment by oSoMoN** [comment]
Thanks @Mejans, I generated the files for the translation to be available in the application, and I merged your branch.

**Pull request merged**